python/src/spark_rapids_ml/classification.py

```diff
@@ -825,6 +844,22 @@ def _logistic_regression_fit(
             )
 
             def _single_fit(init_parameters: Dict[str, Any]) -> Dict[str, Any]:
+                reg_param = (
```

Comment:
Looks strange to map back to 'spark' params from C and l1_ratio.  

Could you infer the correct value of 'penalty' from already converted C and l1_ratio?